proteus/mprans/RANS3PF2D.h

```diff
@@ -407,9 +407,10 @@ namespace proteus
   public:
     cppHsuSedStress<2> closure;
     const int nDOF_test_X_trial_element,
-      nSpace2=4;
+      nSpace2;
     CompKernelType ck;
     cppRANS3PF2D():
+      nSpace2{4},
```

Comment:
Thanks! Maybe just use nspace2(4) here since nspace2 is just a `const int` not an aggregate type.